.circleci/config.yml.in

```diff
@@ -207,3 +273,13 @@ workflows:
     jobs:
       - circleci_consistency
       {{ workflows(prefix="nightly_", filter_branch="nightly", upload=True) }}
+  docker_build:
+    triggers:
+      - schedule:
+          cron: "0 10 * * 0"
```

Comment:
Reminder that cron jobs don't get access to environment variables in secret contexts. If you want to do jobs with access to them, you have to do the Chronos triggered push trick

Reply:
OMG!

Reply:
looks like circleCI fix this:  

https://circleci.com/workflow-run/7dc8be6a-9d61-4b14-b6cb-7394f32245f1 

the docker build job is triggered by cron and credentials got passed in

Reply:
No, I disagree with your assessment of the job. Look at https://app.circleci.com/jobs/github/pytorch/pytorch/3712495 and expand "Spin up Environment". None of the environment variables mentioned are those from the context https://circleci.com/gh/organizations/pytorch/settings#contexts/2b885fc9-ef3a-4b86-8f5a-2e6e22bd0cfe

Maybe these smoketests don't actually need the secrets from the context, which is why everything is OK. But just FYI.

Reply:
OK, so what I used is environment variable, which is define here: https://circleci.com/gh/pytorch/audio/edit#env-vars ,  it's different than context